ush/config_defaults.sh

```diff
@@ -291,6 +295,8 @@ OBSPATH_NSSLMOSIAC="/public/data/radar/mrms"
 LIGHTNING_ROOT="/public/data/lightning"
 ENKF_FCST="/lfs4/BMC/public/data/grids/enkf/atm"
 FFG_DIR="/public/data/grids/ncep/ffg/grib2"
+SST_ROOT="/lfs4/BMC/public/data/grids/ncep/sst/0p083deg/grib2"
+update_SST="01"
```

Comment:
Agree with Ming that the default value should be update_SST="99"

Also, this make me think that it would be better to change "`update_SST`" to "SST_update_cyc" or "`SST_update_hour`"

Reply:
It does make better sense if we use "SST_update_cyc" or "SST_update_hour" than the original "update_SST". @hu5970 What do you think?

Reply:
I vote for "SST_update_hour". You and I add up to two votes. I think you may go ahead to make the changes 😁

Reply:
Two votes count! Changed it to "SST_update_hour".